Kudu.Core/Deployment/AutoSwapHandler.cs

```diff
@@ -63,15 +60,7 @@ public bool IsAutoSwapOngoing()
                 tracer.Trace("Jwt token is null");
                 return;
             }
-
-            // active deployment is always a success deployment
-            string lastDeploymentId = _deploymentStatusManager.ActiveDeploymentId;
-            if (string.Equals(currentDeploymetId, lastDeploymentId, StringComparison.OrdinalIgnoreCase))
```

Comment:
This check is no longer valid. For the case of subsequence webhook trigger, we perform autoswap after all deployment is done. 

So remove this check.